sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-redis/src/main/java/com/azure/messaging/eventhubs/checkpointstore/redis/JedisRedisCheckpointStore.java

```diff
@@ -74,35 +88,39 @@ public Mono<Void> updateCheckpoint(Checkpoint checkpoint) {
                 .propagate(new IllegalStateException(
                     "Checkpoint is either null, or both the offset and the sequence number are null.")));
         }
-
+        String prefix = prefixBuilder(checkpoint.getFullyQualifiedNamespace(), checkpoint.getEventHubName(), checkpoint.getConsumerGroup());
         String key = keyBuilder(checkpoint.getFullyQualifiedNamespace(), checkpoint.getEventHubName(), checkpoint.getConsumerGroup(), checkpoint.getPartitionId());
-        Map<String, String> checkpointStorageMap = new HashMap<>();
+        JSONObject checkpointInformation = new JSONObject();
```

Comment:
We should be using SerializerAdapter.. more specifically JacksonSerializerAdapter. I believe there are examples in azure-core